rakelib/artifacts.rake

```diff
@@ -783,6 +791,19 @@ namespace "artifact" do
     end
   end
 
+  def build_docker_from_dockerfiles(flavor)
+    env = {
+      "ARTIFACTS_DIR" => ::File.join(Dir.pwd, "build"),
+      "RELEASE" => ENV["RELEASE"],
+      "VERSION_QUALIFIER" => VERSION_QUALIFIER,
+      "BUILD_DATE" => BUILD_DATE,
+      "LOCAL_ARTIFACTS" => LOCAL_ARTIFACTS
+    }
+    Dir.chdir("docker") do |dir|
+      safe_system(env, "make build-from-dockerfiles_#{flavor}")
```

Comment:
`build-from-dockerfiles_#{flavor}` only support `full` right now. It would be nice to add a comment

Reply:
added wolfi and oss